edgexfoundry/edgex-compose
# feat: [Kamakura] Add new ONVIF and USB device services

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-compose/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have opened a PR for the related docs change (if not, why?) **README updated**
  <link to docs PR>


## Testing Instructions
run `make gen no-secty ds-onvif-camera ds-usb-camera`
Verify camera services added and images are version 2.2.0 from docker hub.
